package.json

```diff
@@ -36,7 +36,7 @@
     "@types/node": "12.7.5",
     "@typescript-eslint/eslint-plugin": "2.3.0",
     "@typescript-eslint/parser": "2.3.0",
-    "@zeit/ncc": "^0.20.5",
+    "@vercel/ncc": "^0.36.1",
```

Comment:
any breaking changes between versions we need to watch out for ?

Reply:
no breaking changes mentioned in the releases